src/components/Modal/components/Dialog/Dialog.scss

```diff
@@ -35,6 +35,10 @@ $large-width: rem(980px);
   background: color('white');
   box-shadow: shadow(layer);
 
+  @media (-ms-high-contrast: active) {
+    border: border(transparent);
```

Comment:
For this line, we'll want to use an actual color instead of `transparent`. Basically, there are two ways to make a border show up:

- Use a transparent border by default in regular CSS
- Use a colored border in the `@media` query

Calling for a transparent border with the `@media` query actually displays a transparent border! Which seems weird, but there we are.

<img width="1207" alt="No border visible in high contrast" src="https://user-images.githubusercontent.com/1462085/64654153-31e78380-d3dd-11e9-8cf0-bcda297502a5.png">

(Please ignore the highlighted text; that's macOS and VMWare not playing nice together while I tried to take a screenshot.)

In this case, I think we might want to go with the default foreground text color for High Contrast, which would look like something like this:

```CSS
@media (-ms-high-contrast: active) {
    border: 1px solid ms-high-contrast-color('text');
  }
```

<img width="1162" alt="Solid default border using the foreground color in High Contrast" src="https://user-images.githubusercontent.com/1462085/64654171-41ff6300-d3dd-11e9-8d61-e611c06610aa.png">

There are some [other examples of that usage in the codebase](https://github.com/Shopify/polaris-react/search?q=-ms-high-contrast&unscoped_q=-ms-high-contrast) that might help.

Let me know too if I can help you get set up with test tools!

Reply:
@LauraAubin your hunch was right! The reason we went with the media query was to be declarative about why the transparent border is there. Let's 🔥the media query and replace with a comment instead.